frontend/src/js/components/tenants/TenantCreateForm.tsx

```diff
@@ -0,0 +1,208 @@
+// Copyright 2024 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { Close as CloseIcon } from '@mui/icons-material';
+import { Divider, Drawer, IconButton, TextField } from '@mui/material';
+import { makeStyles } from 'tss-react/mui';
+
+import Api from '@northern.tech/store/api/general-api';
+import { getOrganization } from '@northern.tech/store/organizationSlice/selectors';
+import { addTenant } from '@northern.tech/store/organizationSlice/thunks';
+import { AppDispatch } from '@northern.tech/store/store';
+import { useradmApiUrlv1 } from '@northern.tech/store/usersSlice/constants';
+import validator from 'validator';
+
+import Form from '../common/forms/form';
+import FormCheckbox from '../common/forms/formcheckbox';
+import PasswordInput from '../common/forms/passwordinput';
+import TextInput from '../common/forms/textinput';
+import InfoHint from '../common/info-hint';
+import { HELPTOOLTIPS, MenderHelpTooltip } from '../helptips/helptooltips';
+import { PasswordLabel } from '../settings/user-management/userform';
+
+interface TenantCreateFormProps {
+  onCloseClick: () => void;
+  open: boolean;
+}
+
+const useStyles = makeStyles()(theme => ({
+  tenantTitle: {
+    fontSize: '17px',
+    fontWeight: 700
+  },
+  buttonWrapper: {
+    justifyContent: 'flex-start !important'
+  },
+  devLimitInput: { marginTop: 10, maxWidth: 150, minWidth: 130 },
+  infoCard: {
+    maxWidth: '500px'
+  },
+  helpTooltip: {
+    marginLeft: theme.spacing(9),
```

Comment:
That seems quite a lot (72px) - are you sure about this? I see in similar setups the `margin-left-small` being used - so 15px

Reply:
the "*requried" element is taking space, if smaller margin used - it overlapes with the hint icon